owncloudApp/src/main/java/com/owncloud/android/ui/activity/BiometricActivity.java

```diff
@@ -125,6 +129,8 @@ public void onAuthenticationError(int errorCode, @NonNull CharSequence errString
             @Override
             public void onAuthenticationSucceeded(@NonNull BiometricPrompt.AuthenticationResult result) {
                 super.onAuthenticationSucceeded(result);
+                SharedPreferencesProvider preferencesProvider = new SharedPreferencesProviderImpl(getApplicationContext());
```

Comment:
getContext should be enough, anyway, as we are going to rearchitect this biometric stuff, we can move it forward.
```suggestion
                SharedPreferencesProvider preferencesProvider = new SharedPreferencesProviderImpl(getContext());
```